.tool-versions

```diff
@@ -1,2 +1,2 @@
-ruby 3.2.1
-nodejs 16.17.1
+ruby 3.2.2
+nodejs 18.16.0
```

Comment:
Is this could be the current LTS? (20.10.0), Is WFE works with that?

Reply:
I will try!
Wanted to make it the same version as the website but if it works with the latest why not.